api/graphql/reservations/reservation_serializers/refund_serializers.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+from decimal import Decimal
+
+from django.utils.timezone import get_default_timezone
+
+from api.graphql.base_serializers import PrimaryKeySerializer
+from api.graphql.validation_errors import ValidationErrorCodes, ValidationErrorWithCode
+from merchants.models import OrderStatus, PaymentOrder
+from reservations.models import STATE_CHOICES, Reservation
+from reservations.tasks import refund_paid_reservation_task
+
+DEFAULT_TIMEZONE = get_default_timezone()
+
+
+class ReservationRefundSerializer(PrimaryKeySerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    class Meta:
+        model = Reservation
+        fields = [
+            "pk",
+        ]
+
+    @property
+    def validated_data(self):
+        validated_data = super().validated_data
+        return validated_data
+
+    def validate(self, data):
+        now = datetime.datetime.now(tz=DEFAULT_TIMEZONE)
+        if self.instance.price_net <= Decimal("0.0"):
+            raise ValidationErrorWithCode(
+                "Only reservations with price > 0 can be refunded.",
```

Comment:
I'd change the `price > 0` to `price greater than zero`

Reply:
True, that is more human and less nerdy 🤓